targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F437xG/TARGET_UBLOX_C030/battery_charger_i2c.h

```diff
@@ -0,0 +1,27 @@
+#ifndef I2C_BATT_CHARGER_H
```

Comment:
Suggest make this exactly the same as the filename, so `BATTERY_CHARGER_I2C`.

Reply:
Alright